File: app/src/main/java/cn/nicolite/huthelper/view/iview/IFreshmanGuideView.java
package cn.nicolite.huthelper.view.iview;

import java.util.List;

import cn.nicolite.huthelper.base.iview.IBaseView;
import cn.nicolite.huthelper.model.bean.FreshmanGuide;

/**
 * Created by nicolite on 17-11-13.
 */

public interface IFreshmanGuideView extends IBaseView {
    void showGuideList(List<FreshmanGuide> list);
}

File: app/src/main/java/cn/nicolite/huthelper/view/iview/IFreshmanGuideView.kt
package cn.nicolite.huthelper.view.iview

import cn.nicolite.huthelper.base.IBaseView
import cn.nicolite.huthelper.model.bean.FreshmanGuide

/**
 * Created by nicolite on 17-11-13.
 */

interface IFreshmanGuideView : IBaseView {
    fun showGuideList(list: List<FreshmanGuide>)
}
